case/src/main/kotlin/com/ritense/case/domain/DateFormatDisplayTypeParameter.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2015-2022 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.case.domain
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class DateFormatDisplayTypeParameter(
+    @JsonProperty("dateFormat")
+    private val dateFormat: String
+): DisplayTypeParameter {
+    override fun validate(): Boolean {
+        return dateFormat.isNotEmpty()
```

Comment:
The date format should be optional. It will fall back to the default format when not filled in.

Reply:
you are correct. will fix